new-lamassu-admin/src/components/editableTable/Row.styles.js

```diff
@@ -23,7 +23,8 @@ export default {
     }
   },
   suffix: {
-    marginLeft: 7
+    marginLeft: 7,
+    fontSize: '12px'
```

Comment:
Is there a reason why this is 12px? I'm pretty sure we're using odd numbers for our fonts. Also this should be using something from the typography module instead of hard coding.

Reply:
According to the card's description and the design, the suffix should be 12px and the number 16px.